### PR TITLE
create model, controller and route. (only title field)

### PR DESCRIPTION
**Pull request by OscarHagman**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,5 @@
+.gitignore
```
Comment (ArastoSahbaei) [inline .gitignore:1]: ignore the .gitignore? why?

File: .gitignore [hunk 2]
```diff
@@ -1,3 +1,5 @@
...
+
 # reated by https://www.toptal.com/developers/gitignore/api/node
 # Edit at https://www.toptal.com/developers/gitignore?templates=node
```
Comment (ArastoSahbaei) [inline .gitignore:4]: was is das?

File: package.json [hunk 3]
```diff
@@ -16,6 +16,7 @@
   "license": "ISC",
   "description": "",
   "dependencies": {
+    "body-parser": "^1.19.0",
```
Comment (ArastoSahbaei) [inline package.json:19]: ska inte behövas

**Review by ArastoSahbaei — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (OscarHagman) [reply .gitignore:1]: have some files in my local repository that's in my .gitignore that i don't want to be public, not a big deal really and i can revert it back to normal. don't know why git still uploaded the .gitignore file though

File: .gitignore (hunk 2 above)
Comment (OscarHagman) [reply .gitignore:4]: copy pasted your .gitignore file for node and those lines came with it

**Pull request merged**